src/actions/pkg.cr

```diff
@@ -48,22 +38,31 @@ module DppmRestApi::Actions::Pkg
     end
     raise Unauthorized.new context
   end
+
   # Clean unused built packages
-  relative_delete "/clean" do |context|
+  relative_delete "/:source_name/clean" do |context|
     if Actions.has_access? context, Access::Delete
-      clean_unused_packages(context).to_json context.response
-      context.response.flush
-      next context
+      prefix = get_prefix_with_source_name context
+      result = prefix.clean_unused_packages(confirmation: false) { }
```

Comment:
Has the implementation changed to return an empty set instead of `nil`?

Reply:
Yes, I looked into this and indeed always returning a `Set` instead of an union with `Nil` sounds better.